packages/rx_bloc_cli/bin/compile.sh

```diff
@@ -20,7 +25,8 @@ mkdir example/test_app
 dart run rx_bloc_cli create \
   --project-name=test_app \
   --organisation=com.primeholding \
-  --enable-feature-counter=true \
+  --enable-feature-counter=false \
```

Comment:
Let's enable all features in the compile command. This will help reduce regression over time

Reply:
Done